views/js/controller/creator/templates/section-props.tpl

```diff
@@ -153,7 +153,7 @@
         <div class="category-presets"></div>
     </div>
 
-    <h4 class="toggler closed" data-toggle="~ .section-selection">{{__ 'Selection'}}</h4>
+    <h4 class="toggler closed" data-toggle="~ .section-selection">{{__ 'Selection'}} </h4>
```

Comment:
I suggest to revert this change, if is just a space :) 

```suggestion
    <h4 class="toggler closed" data-toggle="~ .section-selection">{{__ 'Selection'}}</h4>
```

Reply:
fixed!! thanks @gabrielfs7   !